core/testdata/proto/test.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package test;
+
+service TestService {
+  rpc Test (TestRequest) returns (TestReply) {}
+}
+
+message TestRequest {
+  string field1 = 1;
+}
+
+message TestReply {
+  string field1 = 1;
+}
```

Comment:
Missing eol at eof: https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b

Reply:
Thanks, fixed in 46e858a